src/__tests__/lib/exampleData.js

```diff
@@ -112,16 +113,15 @@ export const makeCrossRealmBot = (args: { name?: string } = {}): CrossRealmBot =
 
 export const realm = 'https://zulip.example.org';
 
-export const zulipVersion = '2.1.0-234-g7c3acf4';
+export const zulipVersion = new ZulipVersion('2.1.0-234-g7c3acf4');
 
 export const makeAccount = (
   args: {
     user?: User,
     email?: string,
     realm?: string,
     apiKey?: string,
-    shouldHaveZulipVersion?: boolean,
-    zulipVersion?: string,
+    zulipVersion?: ZulipVersion | null,
```

Comment:
I believe `args` is now of type `$Shape<Account>`, and can be annotated thus.

Reply:
I wish it were so, for simplicity's sake, but `user` isn't in `Account`.